src/sources/GeoJSON.js

```diff
@@ -58,6 +58,8 @@ export default class GeoJSON extends Base {
         }
 
         this._features = this._initializeFeatureProperties(this._features);
+
+        this._setCoordinatesCenter();
```

Comment:
I would try to be more explicit here: 'this._dataframeCenter = this.coordinatesCenter();`